mazda/callbacks.cpp

```diff
@@ -261,6 +264,48 @@ std::string MazdaCommandServerCallbacks::GetLogPath() const
     return "/tmp/mnt/data/headunit.log";
 }
 
+void AudioManagerClient::aaRegisterStream()
+{
+	if(!aaStreamRegistered)
+	{
+		// so we dont accedentally try to register this twice
+		aaStreamRegistered = true;
+		try
+		{
+			// First open a new Stream
+			json sessArgs = {
+				{ "busName", "com.xsembedded.ServiceProvider" },
+				{ "objectPath", "/com/xse/service/AudioManagement/AudioApplication" },
```

Comment:
@lmagder would you happen to know what the obectPath should be? I think its the final piece of this puzzle